src/frontend/src/pages/ChangeRequestDetailPage/OtherChangeRequestsPopupTabs.tsx

```diff
@@ -113,9 +109,12 @@ const OtherChangeRequestsPopupTabs: React.FC<OtherChangeRequestsPopupTabsProps>
           mb: '-1px'
         }}
       >
-        {displayTab(1, `Other CR's from ${fullNamePipe(changeRequest.submitter)}`)}
+        {displayTab(1, `Other CR's on ${wbsPipe(changeRequest.wbsNum)}`)}
```

Comment:
can you add the wbsName here like how it looks inside the cards that pop up when u click the tab

Reply:
it looks wacky, needs to be on two lines for that to work

Reply:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/102765855/f33dba8f-7c5e-47a0-b952-212477ac9a0b)